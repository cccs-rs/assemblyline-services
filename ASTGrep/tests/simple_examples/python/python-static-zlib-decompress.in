import zlib
from zlib import decompress

zlib.decompress(b'x\x9cKLL\x04\x00\x02I\x01$')
decompress(b"x\x9cKLL\x04\x00\x02I\x01$")