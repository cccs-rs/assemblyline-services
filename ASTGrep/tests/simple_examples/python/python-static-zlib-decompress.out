import zlib
from zlib import decompress

b'aaa'
b'aaa'